### PR TITLE
Change server exit popup wording

### PR DESCRIPTION
**Pull request by Atlante45**
Make the question in the popup be the last thing just above the answers.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 01aa79758b9832664260adf6aefa0a6c718033d7 available at: https://gist.github.com/475c2ab6007cbbb2bff8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6931.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6931.dmg

File: server-console/src/main.js [hunk 1]
```diff
@@ -110,8 +110,8 @@ function shutdown() {
             dialog.showMessageBox({
                 type: 'question',
                 buttons: ['Yes', 'No'],
-                title: 'Are you sure?',
-                message: 'Quitting will stop your Server Console and your Home domain will no longer be running.'
+                title: 'Stopping Server Console',
+                message: 'Quitting will stop your Server Console and your Home domain will no longer be running.\n Do you wish to continue?'
```
Comment (huffman) [inline server-console/src/main.js]: I think the space after the newline should be removed unless it was done on purpose.

SS from Win10: http://prntscr.com/9v3sx0

**Comment by huffman** [comment]
Looks good other than my comment

File: server-console/src/main.js (hunk 1 above)
Comment (Atlante45) [inline server-console/src/main.js]: Good point

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1f44222ed26f2fce907231c44f0d20a7bdeb68e1 available at: https://gist.github.com/475c2ab6007cbbb2bff8<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6931.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR6931.dmg

**Pull request merged**